monkey/infection_monkey/network/relay/utils.py

```diff
@@ -51,18 +54,14 @@ def _check_if_island_server(server: str) -> bool:
     logger.debug(f"Trying to connect to server: {server}")
 
     try:
-        requests.get(  # noqa: DUO123
-            f"https://{server}/api?action=is-up",
-            verify=False,
-            timeout=MEDIUM_REQUEST_TIMEOUT,
-        )
+        _ = IslandApiClient(server)
```

Comment:
Why not just do this?
```suggestion
        IslandApiClient(server)
```

Reply:
🤷‍♂️